src/guide/kuzzle-backend-setup/aws-marketplace.md

```diff
@@ -4,4 +4,63 @@ algolia: true
 title: AWS Marketplace
 description: AWS Marketplace
 order: 1200
----
\ No newline at end of file
+---
+
+# AWS Marketplace
+<div class="alert alert-info">
+To follow this tutorial, you need a __valid AWS account__.
+</div>
+
+_In this guide, you'll learn where and how to use our AWS Marketplace AMI. It's a good way to test Kuzzle in a cloud environment. In addtion, we recommend you to use our [Kuzzle Admin Console](http://console.kuzzle.io), the most comfy way to play with Kuzzle._
+
+## Get the AMI
+
+Our AMI is stored on AWS Marketplace. It's setup with:
+  * Ubuntu (__16.04__)
+  * Kuzzle (__latest__) with MQTT protocol support.
+  * Elasticsearch (__v5.4.1__).
+  * Redis (__v3.2.12__).
+
+Go to the marketplace and type __kuzzle__ in the search form.
+Choose your Amazon EC2 instance type (the minimal requirement is a __t2-medium__).
+Recover the public IP or the hostname provided by AWS before continue.
+Check if Kuzzle is up and running by simply performing the following HTTP request::
+
+```sh
+$ curl http://yourInstanceIpOrHostname:7512/_serverInfo\?pretty
+{
+  "requestId": "9b07a095-7143-49a5-9079-a34e937fdf3e",
+  "status": 200,
+  "error": null,
+  "controller": "server",
+  "action": "info",
+  "collection": null,
+  "index": null,
+  "volatile": null,
+  "result": {
+    # Some stuff
+  }
+}
+```
+
+You should see some informations about your Kuzzle Server.
+If not, wait a few minutes and retry the request.
```

Comment:
Isn't there a way to check whether Kuzzle is up or not, using AWS?

Reply:
With the special setup I've talk about (no anonymous user). I don't know how... In the AWS console you can only see if the instance is up. If you want to check the logs you can go to the CloudWatch section but it's not the easiest/fastest way to check if Kuzzle is properly running.